cli/command/system/info.go

```diff
@@ -222,9 +223,11 @@ func prettyPrintInfo(dockerCli *command.DockerCli, info types.Info) error {
 	fprintfIfNotEmpty(dockerCli.Out(), "No Proxy: %s\n", info.NoProxy)
 
 	if info.IndexServerAddress != "" {
-		u := dockerCli.ConfigFile().AuthConfigs[info.IndexServerAddress].Username
-		if len(u) > 0 {
-			fmt.Fprintf(dockerCli.Out(), "Username: %v\n", u)
+		if dockerCli.ConfigFile() != nil {
```

Comment:
If this is just because of the test, I think you can revert this. Using the `FakeCli` should give you a default config file I believe.